dhall/src/Dhall/Import.hs

```diff
@@ -1157,11 +1161,27 @@ loadWith expr₀ = case expr₀ of
 
   ImportAlt a b -> loadWith a `catch` handler₀
     where
-      handler₀ (SourcedException (Src begin _ text₀) (MissingImports es₀)) =
-          loadWith b `catch` handler₁
+      is :: forall e . Exception e => SomeException -> Bool
+      is exception =
+          not (null (Exception.fromException @e exception))
```

Comment:
This should be a bit clearer. Initially I was wondering whether there are lists involved here.

```suggestion
          isJust (Exception.fromException @e exception)
```